integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/search/SearchQueryBaseIT.java

```diff
@@ -130,7 +163,8 @@ public void target_byClass_multipleTypes() {
 
 			backendMock.expectSearchObjects(
 					Arrays.asList( Book.NAME, Author.NAME ),
-					b -> { },
+					b -> {
+					},
```

Comment:
Please select the code you want to reformat before launching the automatic formatter next time, this will avoid this kind of noise :)